music21/stream/tests.py

```diff
@@ -4027,6 +4027,8 @@ def procCompare(srcOffset, srcDur, dstOffset, dstDur, divList):
                 n = note.Note()
                 n.quarterLength = srcDur[i]
                 s.insert(srcOffset[i], n)
+            # Must be sorted for quantizing to work optimally.
+            s.sort()
```

Comment:
tiny bit confused by this addition -- Streams are autosorted by default.  This seems to be a problem of quantize if it somehow bypasses the normal "sort before iterate/access" paradigm.

Reply:
Might have been an artifact from an earlier commit. Does look unnecessary now. 😅